components/browser/search/src/main/java/mozilla/components/browser/search/ext/SearchEngineManager.kt

```diff
@@ -18,14 +19,14 @@ import mozilla.components.browser.search.SearchEngineManager
  *
  * https://github.com/mozilla-mobile/android-components/issues/8686
  */
-fun SearchEngineManager.toDefaultSearchEngineProvider(
+fun LazyComponent<SearchEngineManager>.toDefaultSearchEngineProvider(
```

Comment:
I wouldn't change search right now as @pocmo has been working on a big refactoring which is about to land in Fenix.